tests/performance/test_data_analysis/statistical_data_analyzer.py

```diff
@@ -0,0 +1,274 @@
+        """
+        output = ""
+
+        for process in self.processes:
+            baseline_data = self.baseline[self.baseline[self.process_name] == process]
+            datasource_data = self.datasource[self.datasource[self.process_name] == process]
+
+            for metric in self.metrics:
+                table = PrettyTable()
+                table.title = process + " - " + metric
+                table.field_names = ['Name', 'Mean', 'Max value', 'Min value', 'Standard deviation', 'Variance']
+                table.add_row([
+                    "Baseline",
+                    round(baseline_data[metric].mean(), 2),
+                    baseline_data[metric].max(), baseline_data[metric].min(),
+                    round(baseline_data[metric].std(), 2),
+                    round(baseline_data[metric].var(), 2)
+                ])
+                table.add_row([
+                    "Data source",
+                    round(datasource_data[metric].mean(), 2),
+                    datasource_data[metric].max(), datasource_data[metric].min(),
+                    round(datasource_data[metric].std(), 2),
+                    round(datasource_data[metric].var(), 2)
+                ])
+                output += table.get_string() + "\n\n"
+
+        return output
+
+
+class StatisticalComparator:
+    """Class that contains the necessary methods to perform a comparison between the statistics
+    of a certain process and a metric. The comparison is made around a threshold value which is
+    set in the YML file.
+    """
+
+    def calculate_basic_statistics(self, dataframe: pd.DataFrame, metric: str, stat: str) -> float:
+        """Calculates the basic statistic of a data set for a specific metric.
+
+        Args:
+            dataframe (pd.DataFrame): Dataframe containing the specific data for the calculation.
+            metric (str): Metrics on which the statistic will be calculated.
+            stat (str): statistics to be calculated.
+
+        Returns:
+            value (float): value of the calculated statistic.
+        """
+        value = 0
+        if stat == 'Mean':
+            value = round(float(dataframe[metric].mean()), 2)
+        elif stat == 'Median':
+            value = round(float(dataframe[metric].median()), 2)
+        elif stat == 'Max value':
+            value = round(float(dataframe[metric].max()), 2)
+        elif stat == 'Min value':
+            value = round(float(dataframe[metric].min()), 2)
+        elif stat == 'Standard deviation':
+            value = round(float(dataframe[metric].std()), 2)
+        elif stat == 'Variance':
+            value = round(float(dataframe[metric].var()), 2)
+
+        return value
+
+    def comparison_basic_statistics(self, baseline: pd.DataFrame, datasource: pd.DataFrame, metric: str,
+                                    stat: str, threshold: float) -> int:
+        """Compares the percent change in a statistic between the two data sets to determine
```

Comment:
Same here, the first line should be a summary. Detailed information should be in a description section, separated by one line

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5622/commits/a8be0593badbbd04bb3211468f64ad2689d17bf3